core/server/services/auth/api-key/admin.js

```diff
@@ -0,0 +1,122 @@
+const jwt = require('jsonwebtoken');
+const models = require('../../../models');
+const common = require('../../../lib/common');
+
+const JWT_OPTIONS = {
+    maxAge: '5m',
+    algorithms: ['HS256']
+};
+
+/**
+ * Remove 'Ghost' from raw authorization header and extract the JWT token.
+ * Eg. Authorization: Ghost ${JWT}
+ * @param {string} header
+ */
+const _extractTokenFromHeader = function extractTokenFromHeader(header) {
+    const [scheme, token] = header.split(' ');
+
+    if (/^Ghost$/i.test(scheme)) {
+        return token;
+    }
+
+    return;
+};
+
+const rejectAdminApiKey = function rejectAdminApiKey(req, res, next) {
```

Comment:
Wondering if this should be renamed/moved - it's not really rejecting an admin api key but rather a method of authentication.

Reply:
possibly, but not sure where else it would go, we could in theory just delete it right? if a route doesn't accept that mechanism then it wont read off of it and 401 eventually? I'm guessing this is for developer UX though?

Reply:
Yeah, it's only for developer UX. I think it will be easier/cleaner to delete and add something back in later on if needed - we already have checks for the wrong key type being used so this isn't really adding a lot of benefits.